Applications/ApplicationsLib/UncoupledProcessesTimeLoop.h

```diff
@@ -142,7 +148,7 @@ class UncoupledProcessesTimeLoop
 
     //! Sets and returns initial conditions for the given ProjectData and
     //! PerProcessData.
-    std::vector<Vector> setInitialConditions(
+    void setInitialConditions(
```

Comment:
Since now the method does not return anything the comment above should be changed, too.